bundles/org.openhab.io.homekit/README.md

```diff
@@ -797,6 +797,8 @@ All accessories also support the following optional characteristic that can be l
 | GarageDoorOpener            |                             |                             |                                         | A garage door opener.                                                                                                                                                                                                                                                                                                                                         |                                                                       |                                                                                                             |
 |                             | CurrentDoorState            |                             | Contact, Switch, Number, String         | Current door state.                                                                                                                                                                                                                                                                                                                                           | inverted (false)                                                      | OPEN (0, ON, OPEN), CLOSED (1, OFF, CLOSED), OPENING (2), CLOSING (3), STOPPED (4)                          |
 |                             | TargetDoorState             |                             | Contact, Switch, Number, String         | Target door state.                                                                                                                                                                                                                                                                                                                                            | inverted (false)                                                      | OPEN (0, ON, OPEN), CLOSED (1, OFF, CLOSED)                                                                 |
+|                             |                             | LockCurrentState            | Contact, Number, String, Switch         | Current state of lock mechanism                                                                                                                                                                                                                                                                                                                               | inverted (false)                                                      | UNSECURED (0, OFF, CLOSED), SECURED (1, ON, OPEN), JAMMED (2), UNKNOWN (3)                                  |
```

Comment:
If a lock is `OPEN` i would not sonsider it `SECURED`

Reply:
I'm not going to argue with that, but if we want a one-off change there, it should be a separate change. This is just re-using the same characteristics as the Lock accessory.

Reply:
Ok, most important is that it is consistent, and from your post i understand that it is.